cmake/Modules/Platform/Emscripten.cmake

```diff
@@ -18,7 +18,13 @@ set(CMAKE_SYSTEM_NAME Emscripten)
 set(CMAKE_SYSTEM_VERSION 1)
 
 set(CMAKE_CROSSCOMPILING TRUE)
-set_property(GLOBAL PROPERTY TARGET_SUPPORTS_SHARED_LIBS FALSE)
+
+# shared library (side module) support
+set_property(GLOBAL PROPERTY TARGET_SUPPORTS_SHARED_LIBS TRUE)
+set(CMAKE_SHARED_LIBRARY_CREATE_C_FLAGS "-s SIDE_MODULE=1")    # instead of -shared
```

Comment:
Just `-sSIDE_MODULE` (no space or equals sign)

Reply:
Tested but seems to make no difference. Will adjust nonetheless :)

Reply:
It makes no difference but its more concise and the recommended form these days.